<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_ListUsers</class>
 <widget class="QFrame" name="F_ListUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_12">
   <item>
    <widget class="QLabel" name="L_header">
     <property name="text">
      <string>Add User</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutHeader">
     <item>
      <layout class="QHBoxLayout" name="layoutRowBox">
       <item>
        <widget class="QLabel" name="L_show">
         <property name="text">
          <string>Show</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_rows">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSpinBox" name="Sb_IDUser">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_editRow">
       <property name="text">
        <string>(*)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MDLineEdit" name="W_search" native="true">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>70</height>
        </size>
       </property>
       <property name="labelText" stdset="0">
        <string>Search</string>
       </property>
       <property name="helperText" stdset="0">
        <string/>
       </property>
       <property name="placeHolder" stdset="0">
        <string/>
       </property>
       <property name="helperVisible" stdset="0">
        <bool>false</bool>
       </property>
       <property name="leftIconVisible" stdset="0">
        <bool>false</bool>
       </property>
       <property name="RIconPixmap" stdset="0">
        <pixmap resource="../icons.qrc">:/b/icons/b/ic_search_72pt_2x.png</pixmap>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layoutInputs">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="MDLineEdit" name="W_firstname" native="true">
          <property name="labelText" stdset="0">
           <string>First Name</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_lastname" native="true">
          <property name="labelText" stdset="0">
           <string>Last Name</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_username" native="true">
          <property name="labelText" stdset="0">
           <string>User Name</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_email" native="true">
          <property name="labelText" stdset="0">
           <string>Email</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_company" native="true">
          <property name="labelText" stdset="0">
           <string>Company</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_awardsPoint" native="true">
          <property name="labelText" stdset="0">
           <string>Awards Pionts</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_group" native="true">
          <property name="labelText" stdset="0">
           <string>Group Users</string>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="MDLineEdit" name="W_status" native="true">
          <property name="labelText" stdset="0">
           <string>Status</string>
          </property>
          <property name="helperText" stdset="0">
           <string/>
          </property>
          <property name="placeHolder" stdset="0">
           <string/>
          </property>
          <property name="helperVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="leftIconVisible" stdset="0">
           <bool>false</bool>
          </property>
          <property name="rightIconVisible" stdset="0">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layoutSubmit">
        <item>
         <widget class="QPushButton" name="Bt_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bt_cancel">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>400</height>
      </size>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>0</number>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFooter">
     <item>
      <widget class="QLabel" name="L_info">
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPagesCTR">
       <item>
        <widget class="QPushButton" name="Bt_previous">
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="Cb_pages"/>
       </item>
       <item>
        <widget class="QPushButton" name="Bt_next">
         <property name="text">
          <string>&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MDLineEdit</class>
   <extends>QWidget</extends>
   <header>mdlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
